store/localstore/kv_test.go

```diff
@@ -465,11 +465,11 @@ func (s *testKVSuite) TestConditionIfEqual(c *C) {
 	for i := 0; i < cnt; i++ {
 		go func() {
 			defer wg.Done()
-			txn, err = s.s.Begin()
-			c.Assert(err, IsNil)
-			txn.Set(b, []byte("newValue"))
-			err = txn.Commit()
-			if err == nil {
+			mtxn, merr := s.s.Begin()
```

Comment:
Right, we can extend the coding style https://github.com/golang/go/wiki/CodeReviewComments in contribution document.